Library/Formula/sngrep.rb

```diff
@@ -0,0 +1,23 @@
+class Sngrep < Formula
+  homepage "https://github.com/irontec/sngrep"
+  url "https://github.com/irontec/sngrep/archive/v0.2.2.tar.gz"
```

Comment:
Thanks for popping by to clarify! If it's just a couple of days, the maintainers are probably happy to wait on the release to go stable. A couple of days won't hurt.